README.md

```diff
@@ -129,7 +129,7 @@ For **Nuxt.js 2**:
 module.exports = {
   // ...
   build: {
-    transpile: [/^vue-awesome/]
+    transpile: [/^vue-awesome\//]
```

Comment:
It should be `[\\/]` as Windows doesn't support `/`.

PS. Would you mind update `README.zh_CN.md` as well? Thanks!